components/git/clone/component.yaml

```diff
@@ -0,0 +1,18 @@
+name: Git clone
+description: Creates a shallow clone of the specified repo branch
+inputs:
+- {name: Repo URI, type: URI}
+- {name: Branch, type: String, default: master}
+outputs:
+- {name: Repo dir, type: Directory}
+implementation:
+  container:
+    image: alpine/git
+    command:
+    - git
+    - clone
+    - --depth=1
```

Comment:
Shall we consider making depth an input parameter?

Reply:
I'm not sure about that.
The idea is to export the data, rather than creating a full clone. (Maybe I need to rename the component.)
Even the `.git` subdirectory should probably be deleted.
WDYT?

Reply:
Actually I don't have a strong opinion. Either way looks fine. What's the example use case of this component?